wallet/invoices.c

```diff
@@ -219,12 +219,13 @@ bool invoices_load(struct invoices *invoices)
 const struct invoice *invoices_create(struct invoices *invoices,
 				      u64 *msatoshi TAKES,
 				      const char *label TAKES,
-				      u64 expiry)
+				      u64 expiry,
+				      char *b11enc,
+				      struct preimage r,
+				      struct sha256 rhash)
```

Comment:
Minor nit: prefer to hand as 'const struct preimage *r` and `const struct sha256 *rhash` please.  We don't need a copy here.